src/js/htmlutils.js

```diff
@@ -143,11 +145,16 @@ var htmlUtils = exports.htmlUtils = {
 
     // If origin has been whitelisted set text for DNT.
     var whitelistedText = '';
+    var actionDescription = '';
     if (isWhitelisted) {
       whitelistedText = '' +
         '<div id="dnt-compliant">' +
         '<a target=_blank href="https://www.eff.org/privacybadger#faq--I-am-an-online-advertising-/-tracking-company.--How-do-I-stop-Privacy-Badger-from-blocking-me?">' +
-        '<img src="/icons/dnt-16.png" title="This domain promises not to track you."></a></div>';
+        '<img src="/icons/dnt-16.png"></a></div>';
+
+      actionDescription = i18n.getMessage('dnt_tooltip');
```

Comment:
We should make this logic part of `getActionDescription`, since it seems like the logic belongs there conceptually.